src/login/refresh-token.jsx

```diff
@@ -0,0 +1,11 @@
+const refreshToken = async(handle, refreshTiming) => {
```

Comment:
Same as before: This file should have the `.js` extension.

Reply:
Done